### PR TITLE
Blox 187 fe page layout component

### PR DESCRIPTION
**Pull request by MilosMarkoni**

### CODE REVIEW

File: packages/ui/src/Layouts/PageLayoutHeader.tsx [hunk 1]
```diff
@@ -32,18 +47,48 @@ export const PageLayoutHeader = React.forwardRef(
...
+  text-decoration: none;
+  color: white;
+  display: flex;
+  height: 64px;
+  align-items: center;
+  justify-content: center;
+  border-bottom: 2px solid transparent;
+
+  &.active {
+    border-bottom: 2px solid ${({ theme }) => theme.activeHeaderItemBorderColor};
```
Comment (aleksaToljic) [inline packages/ui/src/Layouts/PageLayoutHeader.tsx]: we should leave this part if gradient is not supported, but we should have this code (as per design):

```css
border-image-source: linear-gradient(270deg, #90ABC8 0%, #DAE3ED 100%);
border-image-slice: 1;
```

And also add //TODO@all add animations when border is appeared

Comment (aleksaToljic) [reply packages/ui/src/Layouts/PageLayoutHeader.tsx]: image

File: packages/ui/src/Layouts/PageLayoutHeader.tsx [hunk 2]
```diff
@@ -2,10 +2,16 @@
...
 import React from 'react'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components'
 
-import { Text } from '../Text'
+import { Fonts } from '../styles/fonts'
 
-import { BoxLayout } from './BoxLayout'
+type NavItemType = {
+  to: string
```
Comment (aleksaToljic) [inline packages/ui/src/Layouts/PageLayoutHeader.tsx:11]: We should have comments here and maybe switch to `interface NavigationItem` No need to add `Type` or `T` in type name.

[**_Interface work better with objects and method objects, and types are better to work with functions, complex types, etc._**](https://blog.logrocket.com/types-vs-interfaces-in-typescript/)

Comment (MilosMarkoni) [reply packages/ui/src/Layouts/PageLayoutHeader.tsx:11]: Good to know! :)

File: packages/ui/src/Layouts/PageLayoutHeader.tsx (hunk 1 above)
Comment (MilosMarkoni) [reply packages/ui/src/Layouts/PageLayoutHeader.tsx]: Updated! :)

File: packages/ui/src/Layouts/PageLayoutHeader.tsx (hunk 2 above)
Comment (MilosMarkoni) [reply packages/ui/src/Layouts/PageLayoutHeader.tsx:11]: Updated! :)

**Review by aleksaToljic — APPROVED** [review]

**Pull request merged**